frontend/src/app.tsx

```diff
@@ -47,6 +48,26 @@ export function LoggedInRoute<T extends DefaultParams = DefaultParams>({
   return <Route {...props} />;
 }
 
+export function LoggedInRedirectRoute<T extends DefaultParams = DefaultParams>({
+  ...props
+}: RouteProps<T>) {
+  const [_, navigate] = useLocation();
+  const { isLoggedIn, isLoading } = useAuthData();
+  const { data: config, isLoading: configLoading } = useGetConfig({});
+  if (isLoading) {
+    return <Route component={LoadingPage} />;
+  }
+  if (!isLoggedIn && !configLoading) {
+    if (config.logged_out_redirect_url) {
+      window.location.replace(config.logged_out_redirect_url);
```

Comment:
Souldnt this be assign, so the user can go back?

Reply:
If we use assign and the user gos back, they are landing in an infinite loop. That would be way worse ux imo